### PR TITLE
Fix issue with the '@' prefix not being added appropriately in Mention function

### PR DESCRIPTION
**Pull request by nickludlam**
On a user profile page, if you select "Mention" from the top right menu, it incorrectly misses out the '@' prefix for the full username. I've added it, plus changed the comments in UserActions to be clearer about what is happening.

### CODE REVIEW

File: Mammoth/Utils/UserActions.swift [hunk 1]
```diff
@@ -132,7 +132,7 @@ struct UserActions {
         let vc = NewPostViewController()
         vc.isModalInPresentation = true
         vc.fromPro = true
-        vc.proText = "\(user.remoteFullOriginalAcct) "
+        vc.proText = "@\(user.remoteFullOriginalAcct) LETS GOOOO"
```
Comment (bnolens) [inline Mammoth/Utils/UserActions.swift:135]: Probably didn't want to commit LETS GOOOO?

Comment (bnolens) [inline Mammoth/Utils/UserActions.swift:135]: Probably didn't want to commit LETS GOOOO?

Comment (nickludlam) [reply Mammoth/Utils/UserActions.swift:135]: Haha oops yes! I'll fix that

**Comment by nickludlam** [comment]
Updated with the right changes!

**Pull request merged**